web/src/utils/index.tsx

```diff
@@ -0,0 +1,7 @@
+import Divider from '@mui/material/Divider'
+import React, { ReactElement } from 'react'
+
+export const join = <T, U>(items: T[], separator: (index: number) => U): (T | U)[] =>
+  items.flatMap((item, index) => [item, separator(index)]).slice(0, -1)
+
+export const withDividers = (items: ReactElement[]): ReactElement[] => join(items, index => <Divider key={index} />)
```

Comment:
there is a suggestion from the documentation, I think it is better to implement as they are suggesting `If you're using the Divider to wrap other elements, such as text or chips, we recommend changing its rendered element to a plain <div> using the component prop, and setting role="presentation". This ensures that it's not announced by screen readers while still preserving the semantics of the elements inside it.`